### ISSUE TITLE
How does the loss in asr be calculate? 

### ISSUE THREAD

**Issue by xiechen0692** [espnet/espnet#1319]
# Compute the loss at this time step and accumulate it
        if self.ngpu == 0:
            loss = self.model(*x).mean() / self.accum_grad

Hi, I dont understand the code in asr.py. Why can the loss result be calculate directly from model?
Thank you!
